docs-2.0/nebula-cloud/5.solution/5.3.role-and-authority-management.md

```diff
@@ -0,0 +1,25 @@
+# Roles and authority management
+
+The role on the Cloud is different from that of the Nebula Graph. For more information, see [Roles in Cloud Solution](../4.user-role-description.md).
+
+|Roles in Cloud|Roles in Nebula Graph|
+|:---|:---|
+|OWNER|-|
+|ROOT|ROOT|
+|USER|ADMIN/DBA/GUEST/USER|
+
+## Root Management
+
+Only users with **OWNER** authority can manage **ROOT** users.
```

Comment:
**OWNER** is a role. 
And if you use the bold format for roles, use it all the time. The next sentence does not have it.